central/risk/datastore/internal/search/searcher_impl.go

```diff
@@ -21,7 +23,8 @@ var (
 		Reversed: true,
 	}
 
-	riskSACSearchHelper = sac.ForResource(resources.Risk).MustCreateSearchHelper(mappings.OptionsMap)
+	riskSACSearchHelper         = sac.ForResource(resources.Risk).MustCreateSearchHelper(mappings.OptionsMap)
+	riskSACPostgresSearchHelper = sac.ForResource(resources.Risk).MustCreatePgSearchHelper(schema.RiskSchema.OptionsMap)
```

Comment:
Shouldn't we use for all `MustCreatePgSearchHelper` invocations the `schema.<Resource>.OptionsMap` instead of `mappings.OptionsMap` ? Within the test, we specifically use the schema one.